ArduPlane/quadplane.cpp

```diff
@@ -1573,7 +1573,7 @@ void SLT_Transition::update()
         if (!in_forced_transition) {
             const bool show_message = transition_state != TRANSITION_AIRSPEED_WAIT || transition_start_ms == 0;
             if (show_message) {
-                gcs().send_text(MAV_SEVERITY_INFO, "Transition started airspeed %.1f", (double)aspeed);
+                gcs().send_text(MAV_SEVERITY_WARNING, "QAssist: Transition started airspeed %.1f", (double)aspeed);
```

Comment:
We do get this message for all normal transitions, I don't think we can issue the warning in that case. We would need to distinguish a assist in the cause of a transition and a "unexpected" assist.

Reply:
can just check motors in UNLIMITED?